labs/SettingsControls/src/SettingsExpander/SettingsExpander.Properties.cs

```diff
@@ -93,6 +102,15 @@ public object Content
         set => SetValue(ContentProperty, value);
     }
 
+    /// <summary>
+    /// Gets or sets the Footer.
+    /// </summary>
+    public object Footer
```

Comment:
Should this be a `UIElement`? I think the guidance would be if you can provide a template than use `object` (i.e. if we had a `FooterTemplate` property as well), otherwise we use `UIElement` to say that it should be some other XAML construct, eh?

Wondering if we need to do the same for `Header`? Or since we're using `ContentPresenter` if we also want to add `HeaderTemplate`/`FooterTemplate` properties?

Reply:
As discussed, I'll move this to UIElement

Reply:
Don't forget about the same for `Header` as well 😉

Reply:
Ah.. looked at that, and then I realized we use the Header and Description are bind the SettingsCard - where we use these properties (both of type object for accessibility purposes (i.e. checking if these are of type string and then use those as the AutomationProperties.Name of the control that is set for the Content).

Should we leave those as is (for now)?